docs/phaser-sdk-demo-websocket-web3.md

```diff
@@ -0,0 +1,72 @@
+---
+id: phaser-sdk-demo-web3-websocket
```

Comment:
we already have a docs page for this, you want to remove from this PR

Reply:
yeh, this is page is for Solidity version of the tiles chain install, do I use the same page of the plugin ?